options/locale/locale_en-US.ini

```
@@ -1407,6 +1407,8 @@ branch.deleted_by = Deleted by %s
 branch.restore_success = Branch '%s' has been restored.
 branch.restore_failed = Failed to restore branch '%s'.
 branch.protected_deletion_failed = Branch '%s' is protected. It cannot be deleted.
+branch.download=Download Branch „%s“
```

```suggestion
branch.download = Download Branch '%s'
```
I think it'd be better to be consistent with how messages regarding branches refer to the branch names